basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,164 @@
 const helpers = require('./project-1');
 
 // start testing!
+
+test('multiply by Ten', () => {
+    const input = 5;
+    const expected = 50;
+    expect(helpers.multiplyByTen(input)).toEqual(expected);
+});
+
+test('subtract by five', () => {
+    const input = 10;
+    const expected = 5;
+    expect(helpers.subtractFive(input)).toEqual(expected);
+});
+
+test('are by same length', () => {
+    const input = 'toy';
+    const expected = 'dog';
+    expect(helpers.areSameLength(input, expected)).toBeTruthy();
+});
+
+test('are equal', () => {
+    const input = 10;
+    const expected = 10;
+    expect(helpers.areEqual(input, expected)).toBeTruthy();
+});
+
+test('less than ninety', () => {
+    const input = 100;
+    const expected = false;
+    expect(helpers.lessThanNinety(input, expected)).toBe(expected);
+});
+
+test('greater than fifty', () => {
+    const input = 100;
+    const expected = true;
+    expect(helpers.greaterThanFifty(input, expected)).toBe(expected);
+});
+
+test('add', () => {
+    const input = 2;
+    const inputTwo = 3;
+    const expected = 5;
+    expect(helpers.add(input, inputTwo)).toBe(expected);
+});
+
+test('subtract', () => {
+    const input = 3;
+    const inputTwo = 2;
+    const expected = 1;
+    expect(helpers.subtract(input, inputTwo)).toBe(expected);
+});
+
+test('divide', () => {
+    const input = 6;
+    const inputTwo = 2;
+    const expected = 3;
+    expect(helpers.divide(input, inputTwo)).toBe(expected);
+});
+
+test('multiply', () => {
+    const input = 6;
+    const inputTwo = 2;
+    const expected = 12;
+    expect(helpers.multiply(input, inputTwo)).toBe(expected);
+});
+
+test('get Remainder', () => {
+    const input = 6;
+    const inputTwo = 3;
+    const expected = 0;
+    expect(helpers.getRemainder(input, inputTwo)).toBe(expected);
+});
+
+test('is Even', () => {
```

Comment:
Would be good to write a test for this one that will fail as well, always try to test for both possible outcomes.